Probesys/bileto
# imp: Redesign the tickets navigation

## Related issue(s)



https://github.com/Probesys/bileto/issues/229

## Changes



- move tickets navigation in the layout sidebar
- use standard CSS components and utilities to design the navigation
- extract the sidebar in a dedicated file
- remove the `pages/tickets-index.css` file

## How to test manually



- go to the global tickets view and to an organization tickets view → check the sidebar shows and works correctly

## Checklist
